postcss.config.js

```diff
@@ -26,7 +33,7 @@ const tw = {
 };
 
 const config = {
-  // add postcss plugins, required to extend apollo tailwind config
+  // Add postcss plugins, required to extend apollo tailwind config
   plugins: [require('tailwindcss')(tw)],
```

Comment:
Instead of creating a new variable what if we just passed the object directly to the plugin?
```suggestion
  plugins: [
    require('tailwindcss')({
      ...tailwindConfig,
      /**
        * Content array includes any file containing Tailwind utility classes.
        * See: https://tailwindcss.com/docs/content-configuration
        */
      content: ['./src/**/*.{html,js}'],
      darkMode: 'media',
      theme: {
        ...tailwindConfig.theme,
        textColor: {
          // Extending existing text colors below
          ...tailwindConfig.theme.textColor,
          // Overriding existing `brand-new` color below
          'brand-new': {
            DEFAULT: '#2A669F',
            50: '#E4F7F8',
          },
        },
      },
    }),
  ],
```

Reply:
I wanted to add it in a new variable to separate the config from the plugins. will not be making this change